### ISSUE TITLE
Large Text Size

### ISSUE THREAD

**Issue by awieckert** [nss-evening-cohort-7/chatty-turophobia#22]

**Comment by awieckert** [comment]
# Story 
As a user I need to be able to increase the font size.

# Acceptance Criteria
**When** the page loads
**Then** there should be a checkbox for "large font"
**And** when it's checked the font size will increase.
**And**  when it's unchecked the font size will go back to default

# Tech Notes
Event will be added in themeEvents.js

increaeFontSize function can be defined in themeEvents.js

```
const addLargeTextEvent = () => {
  checkBox.addEventListener('change', increaseFontSize);
};
```

**Comment by RitaSyed** [comment]
Closes #63

**Comment by RitaSyed** [comment]
Closes #63
